catalog.html

```diff
@@ -0,0 +1,302 @@
+<p class="card__name">Cat Energy SLIM 500 г</p>
+    <div class="card__row">
+<span class="card__col">Масса</span>
+    <span class="card__col">500 г</span>
+</div>
+<div class="card__row">
+     <span class="card__col">Вкус</span>
+    <span class="card__col">Рис</span>
+</div>
+<div class="card__row">
+    <span class="card__col">Цена</span>
+    <span class="card__col">700 Р.</span>
+</div>
+</div>
+</div>
+<button class="card__button button">Заказать</button>
+</div>
+        </li>
+    </ul>
+    </section>
+
+    <section class="main-catalog-more">
+        <button class="main-catalog-more__button button-more"></button>
+        <h1 class="main-catalog-more__title">Показать еще 100500 товаров</h1>
+        <p class="main-catalog-more__subtitle">На самом деле вкусов гораздо больше!</p>
+        <button class="main-catalog-more__button button button-all">Показать все</button>
+    </section>
+
+    <section class="main-catalog-optionals optionals">
+        <h1 class="main-catalog-optionals__title">Дополнительные товары</h1>
+        <ul class="optionals-list">
+            <li class="optionals-list__item">
+                <span class="optionals-list__title">Сахарозаменитель</span>
+                <div class="optionals-list__container">
+                    <p class="optionals-list__row">1 упаковка (100г)</p>
+                    <p class="optionals-list__row">200 Р.</p>
+                    </div>
+                    <button class="optionals-list__button button">Заказать</button>
+            </li>
+            <li class="optionals-list__item">
+                <span class="optionals-list__title">Питьевая вода</span>
+                <div class="optionals-list__container">
+                    <p class="optionals-list__row">5 литров</p>
+                    <p class="optionals-list__row">50 Р.</p>
+                    </div>
+                    <button class="optionals-list__button button">Заказать</button>
+            </li>
+            <li class="optionals-list__item">
+                <span class="optionals-list__title">Молоко</span>
+                <div class="optionals-list__container">
+                    <p class="optionals-list__row">1 литр</p>
+                    <p class="optionals-list__row">100 Р.</p>
+                    </div>
+                    <button class="optionals-list__button button">Заказать</button>
+            </li>
+            <li class="optionals-list__item">
+                <span class="optionals-list__title">Витамины</span>
+                <div class="optionals-list__container">
+                    <p class="optionals-list__row">1 упаковка (30г)</p>
+                    <p class="optionals-list__row">300 Р.</p>
+                    </div>
+                    <button class="optionals-list__button button">Заказать</button>
+            </li>
+        </ul>
+    </section>
+    
+    <section class="main-catalog-bonus bonus-img main-catalog-bonus--background">
```

Comment:
```suggestion
    <section class="catalog-bonus">
```
куда столько классов? классы пишутся для стилей
ты их написала без надобности